### PR TITLE
Add some quickstart docs

### PR DESCRIPTION
**Pull request by intiocean**
to make it easy for someone to pick up the library and get going
straight away

### CODE REVIEW

File: docs/get_started.rst [hunk 1]
```diff
@@ -23,6 +23,33 @@ Installation
 
         url(r'^select2/', include('django_select2.urls')),
 
+Quickstart
+----------
```
Comment (codingjoe) [inline docs/get_started.rst]: Please add a newline here.

File: docs/get_started.rst [hunk 2]
```diff
@@ -23,6 +23,33 @@ Installation
...
+Here is a quick example to get you started.
```
Comment (codingjoe) [inline docs/get_started.rst]: `s/started./started:/`

File: docs/get_started.rst [hunk 3]
```diff
@@ -23,6 +23,33 @@ Installation
...
+
+0. Follow the installation instructions above
```
Comment (codingjoe) [inline docs/get_started.rst]: `s/above/above./`

File: docs/get_started.rst [hunk 4]
```diff
@@ -23,6 +23,33 @@ Installation
...
+
+1. Add a select2 widget to the form. For example if you wanted Select2 with multiselect you would use
+`Select2MultipleWidget`
```
Comment (codingjoe) [inline docs/get_started.rst]: You have to use double ticks.
```
s/`/``/g
```

File: docs/get_started.rst [hunk 5]
```diff
@@ -23,6 +23,33 @@ Installation
...
+Replacing::
+
+        class MyForm(forms.Form):
```
Comment (codingjoe) [inline docs/get_started.rst]: Please use 4 space indentation.

File: docs/get_started.rst [hunk 6]
```diff
@@ -23,6 +23,33 @@ Installation
 
         url(r'^select2/', include('django_select2.urls')),
 
+Quickstart
```
Comment (codingjoe) [inline docs/get_started.rst]: `s/Quckstart/Quick Start/`

File: docs/get_started.rst [hunk 7]
```diff
@@ -23,6 +23,33 @@ Installation
...
+
+        class MyForm(forms.Form):
+          things = ModelMultipleChoiceField(queryset=Thing.objects.all())
+
+with::
+
+        class MyForm(forms.Form):
+          things = ModelMultipleChoiceField(queryset=Thing.objects.all(), widget=Select2MultipleWidget)
+
+2. Add the css to the `head` of your Django template::
```
Comment (codingjoe) [inline docs/get_started.rst]: `s/css/CSS/`

**Comment by intiocean** [comment]
Spell checking tests! Impressive  

@codingjoe All fixed and updated. Tests are now green ✅

**Pull request merged**